RR_manuscript/supplementary_materials.Rmd

```diff
@@ -65,7 +82,7 @@ msa_models_unique <- left_join(
       n_random_slo, n_random_smooth, n_models
       ) %>%
     distinct(), 
-  readRDS("./data/peer_review/msa_reviews.rds") %>% 
+  readRDS(here("./data/peer_review/msa_reviews.rds")) %>% 
```

Comment:
no need for here()

Reply:
Got errors for both the plot and the table with out it. CHange it if you like, but it works as is.

Reply:
It does not work because the setup chunk is not labelled in the `{r}` part but using `#|`. Had this issue with other rmd files

Reply:
I think I'd need to see a reprex to believe that.

Reply:
Mira aquí cabroncito

<img width="865" alt="works" src="https://user-images.githubusercontent.com/6429302/213125348-214b8f47-7f51-404a-bae4-1f762cae5dd4.png">
<img width="866" alt="no-works" src="https://user-images.githubusercontent.com/6429302/213125364-c0c8be47-d8d7-481f-9b22-c285fab2a247.png">

Reply:
So every time you were salty about me not using `here()` was because you were wrong :D WHO HAS THE SALT NOW?! 🧂🧂🧂 Admit your final defeat. There is no `here()` in this dojo.

Reply:
Ok. I will test this later. For now I have put your preferred style of code chunks and it won't knit for me. You can take a look. Pesado.

Reply:
Ah, we had the same issue with the RR when using include_graphics if the figures are not in the same folder as the Rmd. Solved using `error = FALSE`. See here: https://github.com/yihui/knitr/issues/1825